service/src/test/java/de/adorsys/psd2/sandbox/xs2a/ais/AisConsentCreationSteps.java

```diff
@@ -259,6 +323,38 @@ public void receiveAccountData() {
         equalTo(context.getConsentAccountAccess().getAccounts().size()));
   }
 
+  @Then("the transaction list data are received")
+  public void receiveTransactionListData() {
+    ResponseEntity<TransactionsResponse200Json> actualResponse = context.getActualResponse();
+
+    assertThat(actualResponse.getBody().getTransactions().getBooked().size(), equalTo(5));
+    assertThat(actualResponse.getStatusCode().is2xxSuccessful(), equalTo(true));
+  }
+
+  @Then("the transaction data are received")
+  public void receiveTransactionData() {
+    ResponseEntity<KeyedTransactionDetails> actualResponse = context.getActualResponse();
+
+    assertThat(
+        actualResponse.getBody().getTransactionsDetails().getRemittanceInformationStructured()
+            .isEmpty(),
+        equalTo(false));
+    assertThat(actualResponse.getStatusCode().is2xxSuccessful(), equalTo(true));
+  }
+
+  @Then("the balance data are received")
+  public void receiveBalanceData() {
+    ResponseEntity<ReadAccountBalanceResponse200> actualResponse = context.getActualResponse();
+
+    assertThat(actualResponse.getBody().getBalances().size(), equalTo(2));
+    // TODO: XS2A does not show a status for AVAILABLE Types. Responded status is null
+//    assertThat(actualResponse.getBody().getBalances().get(0).getBalanceType(), equalTo(
+//        BalanceType.AVAILABLE));
+    assertThat(actualResponse.getBody().getBalances().get(1).getBalanceType().toString(), equalTo(
+        BalanceType.CLOSINGBOOKED.toString()));
+    assertThat(actualResponse.getStatusCode().is2xxSuccessful(), equalTo(true));
```

Comment:
you already checked that in a previous step

Reply:
Done